org.sonarlint.eclipse.jdt/src/org/sonarlint/eclipse/jdt/internal/JavaProjectConfiguratorExtension.java

```diff
@@ -73,8 +74,8 @@ private static boolean isClassPresentAtRuntime(String className) {
   }
 
   @Override
-  public Set<Language> whitelistedLanguages() {
-    return isJdtPresent() ? SonarLintUtils.STANDALONE_MODE_LANGUAGES_JDT : Collections.emptySet();
+  public Set<SonarLintLanguage> enableLanguages() {
+    return isJdtPresent() ? EnumSet.of(SonarLintLanguage.JAVA, SonarLintLanguage.JSP) : Collections.emptySet();
```

Comment:
Why is it not referring to `SonarLintUtils.OPTIONAL_LANGUAGES`?

Reply:
Because in the future, `SonarLintUtils.OPTIONAL_LANGUAGES` can grow.